ds-streamlit/Jenkinsfile

```diff
@@ -0,0 +1,26 @@
+def odsNamespace = env.ODS_NAMESPACE ?: 'ods'
```

Comment:
this file should be updated as per latest ODS provision design requirements, see for example https://github.com/opendevstack/ods-quickstarters/blob/master/be-python-flask/Jenkinsfile

Reply:
this seems to be a bigger upgrade, as the "create-ds-component.sh" expects nexus, etc.